docker-compose-examples/grist-with-postgres-redis-minio.yml

```diff
@@ -0,0 +1,88 @@
+# This examples shows how to start up Grist that:
+# - Uses Postgres as a home database,
+# - Redis as a state store.
+# - MinIO for snapshot storage
+
+# It is STRONGLY RECOMMENDED not to use this container in a way that makes it accessible to the internet.
+# This setup lacks basic security or authentication.
+# See https://support.getgrist.com for more information.
+
+# This setup is based on one provided by Akito (https://github.com/theAkito).
+
+services:
+  grist:
+    image: gristlabs/grist:latest
+    environment:
+      # Use Python 3 instead of 2.
+      PYTHON_VERSION: 3
+
+      # Postgres database setup
+      TYPEORM_DATABASE: grist
+      TYPEORM_USERNAME: grist
+      TYPEORM_HOST: grist-db
+      TYPEORM_LOGGING: false
+      TYPEORM_PASSWORD: CHANGE THIS PASSWORD
+      TYPEORM_PORT: 5432
+      TYPEORM_TYPE: postgres
+
+      # Redis setup
+      REDIS_URL: redis://grist-redis
+
+      # MinIO setup
```

Comment:
Could maybe have a comment about how the bucket needs to have version control enabled (I see you have code to do this later)

Reply:
Good shout, done!